common/src/limits.rs

```diff
@@ -45,6 +45,7 @@ impl From<AccountTier> for Limits {
             AccountTier::Admin
             | AccountTier::Basic
             | AccountTier::PendingPaymentPro
+            | AccountTier::CancelledPro
             | AccountTier::Deployer => Self::default(),
             AccountTier::Pro | AccountTier::Team => Self::new(MAX_PROJECTS_EXTRA, true),
```

Comment:
I think it should be like this, right? The cancelled pro user should have access until it expires and they are made a Basic user.
```suggestion
            | AccountTier::Deployer => Self::default(),
            AccountTier::Pro | AccountTier::CancelledPro | AccountTier::Team => Self::new(MAX_PROJECTS_EXTRA, true),
```